lib/benchpark/experiment.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Dict, Tuple
+import yaml  # TODO: some way to ensure yaml available
+
+from benchpark.directives import SpecTemplate
+import benchpark.spec
+import benchpark.paths
+import benchpark.repo
+import benchpark.runtime
+import benchpark.variant
```

Comment:
```suggestion
import benchpark.paths
import benchpark.runtime
```
Some of these imports either aren't used or won't be once we delete vestigial code

Reply:
Our style checker should detect these (and generally has as I refactored things). Some of the references are non-obvious, e.g. `import benchpark.variant` is only used by the type enforcement logic, but the import is still needed for that.